package.json

```diff
@@ -74,12 +74,13 @@
     "test": "cross-env NODE_ENV=test node scripts/test.js --env=./scripts/custom-test-env.js --watchAll --coverage",
     "eject": "react-scripts eject",
     "lint:check": "eslint \"**/*.{ts,tsx}\" --max-warnings=0",
-    "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix",
-    "format:fix": "prettier --write \"**/*.{ts,tsx,json,scss,css}\"",
+    "lint:fix": "eslint --fix",
+    "format:fix": "prettier --write",
     "format:check": "prettier --check \"**/*.{ts,tsx,json,scss,css}\"",
```

Comment:
Why these changes are required ?

Reply:
Because lint-staged provides the staged files based on glob patterns specified in .lintstagedrc.json to these commands, So the glob patterns hard-coded in the commands need to be removed.

Image from lint-staged npm page:-
![Screenshot from 2023-12-30 19-18-04](https://github.com/PalisadoesFoundation/talawa-admin/assets/121512095/6f516d83-a070-4c43-8ebf-d51931cc78fd)

Reply:
Got it, thank you.